doc/variableexplorer.rst

```diff
@@ -2,95 +2,213 @@
-Several examples of this functionality follow:
 
 |
 
-.. image:: images/variable_explorer/variable_explorer_text_long.png
+~~~~~~~
+Strings
+~~~~~~~
+
+When a string variable is longer than forty characters, you can double click it to see its value in a text editor where you can modify it.
+
+.. image:: images/variable_explorer/variable-explorer-text-long.png
    :align: center
    :alt: Variable Explorer text editor, displaying a long string in a window
 
 |
 
-.. image:: images/dialog/dialog_user_env_variables_edit.png
+~~~~~~~~~~~~
+Dictionaries
+~~~~~~~~~~~~
+
+When the type of a variable is a dictionary, double clicking it will show you a viewer displaying each one of the keys of the dictionary along with its associated value. You can double click any of the values to modify it. 
+
+.. image:: images/variable_explorer/variable-explorer-dictionary.png
    :align: center
    :alt: Dictionary editor displaying keys and their types, sizes, and values
 
 |
 
-.. image:: images/variable_explorer/variable_explorer_list_timedelta_edit.png
+~~~~~
+Lists
+~~~~~
+
+For variables that are of type list, the window pane only displays the first ten values of the list. If you want to see them all, double click the variable in the pane to open a viewer that will display the index, type, size and value of each element of the list. You can also modify any value by double clicking it in the viewer and writing the new one.
+
+.. image:: images/variable_explorer/variable-explorer-list.png
    :align: center
-   :alt: List editor displaying a list of timedeltas, showing one being edited
+   :alt: List editor displaying a list, showing one being edited
 
 |
 
-.. image:: images/variable_explorer/variable_explorer_array_2D_resize.png
+~~~~~~~~~~~~
+Numpy Arrays
+~~~~~~~~~~~~
+
+For Numpy Arrays, the window pane only shows some of the values of the array. You can open a viewer that shows you graphically in a "heat map" the values of the array, displaying each value in a colored box, depending on the size of the value. You can deactivate the background color by unchecking the option in the window. If the array is too large, background color will be automatically deactivated.
+
+.. image:: images/variable_explorer/variable-explorer-array-2D-resize.png
    :align: center
    :alt: Array editor with a 2D int array, displaying a "heatmap" of its values
 
 |
 
-|varexpcontextmenu| |histogram|
+If possible, you can also change the format of the values in the array, choosing the number of decimals that you want the array to display. For this, click the option format and in the window displayed you can set the desired format, using printf formatting. For more information go to "https://docs.python.org/3/library/stdtypes.html#printf-style-bytes-formatting".
```

Comment:
```suggestion
If possible, you can also change the format of the values in the array, choosing the number of decimals that you want the array to display. For this, click the option format and in the new window that appears you can set the desired format, using ``printf`` formatting. For more information about this, please go to "https://docs.python.org/3/library/stdtypes.html#printf-style-bytes-formatting".
```

Reply:
Please also use a rest link to reference the webpage you're adding in this paragraph.